clr-k8s-examples/README.md

```diff
@@ -3,6 +3,9 @@
 ## Prerequisite
 This setup currently will work with Kubernetes 1.14 & above. Any version of Kubernetes before that might work, but is not guaranteed.
 
+## QUICK NOTE
+If you have an existing kubernetes cluster 1.17 & below on ClearLinux and would like to update to the latest version, please note a direct upgrade would not work. Follow the instructions provided at [k8s-migration](https://docs.01.org/clearlinux/latest/guides/clear/k8s-migration.html) in ClearLinux documentation.
```

Comment:
Suggest: copy the the background section from the migration page and replace the first sentence with it.

Reply:
@mythi  thanks and done.